shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -422,11 +422,11 @@ private static PointerCoords parsePointerCoords(Object rawCoords, float density)
     private void validateVirtualDisplayDimensions(int width, int height) {
         DisplayMetrics metrics = context.getResources().getDisplayMetrics();
         if (height > metrics.heightPixels || width > metrics.widthPixels) {
-            String error = "Creating a virtual display of size: "
+            String message = "Creating a virtual display of size: "
                 +  "[" + width + ", " + height + "]"
                 + " is not supported. It is larger than the device screen size: "
```

Comment:
We should probably update the message(it is supported, up to a limit...)

Reply:
@amirh How can we solve this issue ? And we should use flutter dev or stable ?